chall07/egarcia-.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"io/ioutil"
+	"strings"
+	"strconv"
+	"log"
+)
+
+func main() {
+	response, err := http.Get("https://chall07.42madrid.com/")
+	contents, err := ioutil.ReadAll(response.Body)
+	body := strings.Split(string(contents), " ")
+	string := strings.Split(string(body[0]), ",")
+	red, err := strconv.Atoi(string[1][2:])
+	green, err := strconv.Atoi(string[2][2:])
+	blue, err := strconv.Atoi(string[3][2:])
```

Comment:
A most simple way of doing this would have been by using [fmt.Sscanf](https://golang.org/pkg/fmt/#Sscanf). As you know exactly how the input should be:
```suggestion
	var id, red, green, blue int
	if _, err := fmt.Sscanf(contents, "id=%d,r=%d,g=%d,b=%d - Send your response here: chall07.42madrid.com/?id=<id>&resp=<hex>", &id, &red, &green, &blue); err != nil {
		log.Fatalln(err)
	}
```